ui/component/claimPreviewTile/view.jsx

```diff
@@ -55,6 +55,8 @@ type Props = {
   isLivestreamActive: boolean,
   livestreamViewerCount: ?number,
   swipeLayout: boolean,
+  isHidden: (string) => void,
+  pulse: boolean,
```

Comment:
Nope, that's how it used to be before we've decided to disable the animation. If we add it to all placeholders the homepage starts to lag again because the animation would play on every single tile for each category.